### ISSUE TITLE
Duplication bug on request

### ISSUE THREAD

**Issue by Psy-Virus** [MrTJP/ProjectRed#1189]
Hey there,
first of all, great mod and thanks for porting it to 1.10.2!

There seems to be a duplication bug if you request something. Not all blocks get duplicated but (I guess) a random amount. 

For example I..
request 1471 Coal and get 1899
request 212 Iron and get 276
request 123 Stone and get 123
request 129 Flint and get 129
request 181 Moss Stone and get 234

Server is running on Forge 12.18.3.2316

Do you need any other information? I'm not sure what else I should provide in this case.
If you need the world, I can upload it. 
I didn't find any errors in the logs.

The following mods are installed:

```
antiqueatlas-1.9.4-4.4.4.jar
autosapling-1.3.3-mc1.10.jar
backpack-3.0.1-1.10.2.jar
betterfoliage-mc1.10.2-2.1.5.jar
BiblioCraft[v2.2.4][MC1.10.2].jar
biomesoplenty-1.10.2-5.0.0.2236-universal.jar
bookshelf-1.10.2-1.4.4.347.jar
chameleon-1.10.2-2.3.0.jar
chisel-mc1.10.2-0.0.10.22.jar
codechickencore-1.10.2-2.4.1.102-universal.jar
CodeChickenLib-1.10.2-2.5.9.267-universal.jar
CoFHCore-1.10.2-4.1.2.157-universal.jar
custommobspawner 3.10.1.jar
decocraft-2.4.2_1.10.2.jar
drzhark's mo'creatures mod-10.0.6.jar
dynmap.jar
enchantingplus-1.10.2-4.1.0.115.jar
farseek-1.9.4-2.1.jar
forgemultipart-1.10.2-2.0.0.31-universal.jar
gravestone-1.10.2-graves-1.0.6.jar
gravestone-extended-1.0.4.jar
InventoryTweaks-1.61-58.jar
jehc-1.10.2-1.2.2.jar
jei_1.10.2-3.14.7.419.jar
mantle-1.10.2-1.1.5.jar
MrTJPCore-1.10.2-2.0.0.19-universal.jar
notenoughitems-1.10.2-2.1.3.220-universal.jar
OptiFine_1.10.2_HD_U_D8.jar
Pam's HarvestCraft 1.10.2j.jar
ProjectRed-1.10.2-4.8.3.47-Base.jar
ProjectRed-1.10.2-4.8.3.47-integration.jar
ProjectRed-1.10.2-4.8.3.47-lighting.jar
ProjectRed-1.10.2-4.8.3.47-mechanical.jar
ProjectRed-1.10.2-4.8.3.47-world.jar
resources
Ruins-1.10.2.jar
Ruins-1.10.2-sources.jar
StorageDrawers-1.10.2-3.7.9.jar
StorageDrawersExtras-1.10-1.1.0.jar
streams-1.9.4-0.3.jar
tconstruct-1.10.2-2.6.5.jar
ThermalDynamics-1.10.2-2.0.2.4-universal.jar
ThermalExpansion-1.10.2-5.1.1.93-universal.jar
ThermalFoundation-1.10.2-2.1.0.82-universal.jar
thutcore-3.12.2.jar
thuttech-4.0.10.c.jar
undergroundbiomesconstructs-1.10-1.0.v.19.jar
waila-1.7.0-b3_1.9.4.jar
```

If you think it could be related to another mod, I'll make some other tests without them.

Thank you in advance

:-)

**Comment by Psy-Virus** [comment]
I just tested it without any other mods except the mods Project Red depends on, of course.
Same behaviour.

**Comment by bookerthegeek** [comment]
Steps to reproduce?

**Comment by MrTJP** [comment]
I can reproduce it just fine. Im looking into it

**Comment by MrTJP** [comment]
Fixed with 7054e5eef55f9c950854f75d47d999839f7aab4e. Since this is a major issue, expect a new version later today.

**Comment by Psy-Virus** [comment]
Wow that was fast, I appreciate that!

Thank you very much!
